casper/src/main/scala/coop/rchain/casper/SafetyOracle.scala

```diff
@@ -42,86 +39,73 @@ trait SafetyOracle[F[_]] {
     * @param estimate Block to detect safety on
     * @return normalizedFaultTolerance float between -1 and 1, where -1 means potentially orphaned
     */
-  def normalizedFaultTolerance(blockDag: BlockDag, estimate: BlockMessage): F[Float]
+  def normalizedFaultTolerance(blockDag: BlockDag, estimate: BlockMessage): Float
 }
 
 object SafetyOracle extends SafetyOracleInstances {
   def apply[F[_]](implicit ev: SafetyOracle[F]): SafetyOracle[F] = ev
```

Comment:
I think the `F` here is still needed, for there are many places write in this way: 
```
def hashSetCasper[F[_]: Sync: Capture: ConnectionsCell: TransportLayer: Log: Time: ErrorHandler: SafetyOracle: BlockStore: RPConfAsk](
```